### PR TITLE
Testing posting to slack

### PR DESCRIPTION
**Pull request by DavidJones02**
Testing that we can post to slack using travis

### CODE REVIEW

File: CommitStatusPoster.groovy [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// Using the Jenkins Groovy Post build plugin to execute the following after every build
```
Comment (eamonnonuallain) [inline CommitStatusPoster.groovy]: test

**Comment by DavidJones02** [comment]
In the statuses, E2E-jenkins is the name of the internal jenkins job we are running, the default is the same job just from before the name was configured :)

**Comment by DavidJones02** [comment]
@donal-tobin-sap Please review

**Comment by donal-tobin-sap** [comment]
:+1:

**Pull request merged**